front/app/api/seats/types.ts

```diff
@@ -2,8 +2,9 @@ import { Keys } from 'utils/cl-react-query/types';
 import seatsKeys from './keys';
 
 export type SeatsKeys = Keys<typeof seatsKeys>;
+export type TSeatType = 'collaborator' | 'admin';
 
-export interface SeatsType {
+export interface ISeats {
```

Comment:
For clarification, is this a standard naming way for interfaces within the team?